src/Validator/IsInt.php

```diff
@@ -39,6 +42,13 @@ class IsInt extends AbstractValidator
     protected $locale;
 
     /**
+     * Type of strict check to be used.  "123" == 123
```

Comment:
I think comment could be expanded a little to explain what happens when `"123" == 123` would be evaluated (i.e. say it would fail validation).